app/constants/objects/initializeFoodForm.ts

```diff
@@ -0,0 +1,44 @@
+import { Food } from "../types/types";
+
+export const initializeFoodForm: Readonly<Food> = {
+  uid: "",
+  name: "",
+  description: "",
+  nutritionalData: {
+    calories: 0,
+    protein: {
+      quantity: 0,
+      unit: "g",
+    },
+    fat: {
+      quantity: 0,
+      unit: "g",
+    },
+    carbohydrates: {
+      quantity: 0,
+      unit: "g",
+    },
+    fiber: {
+      quantity: 0,
+      unit: "g",
+    },
+    isGlutenFree: false,
+    isDairyFree: false,
+    isVegetarian: false,
+    isVegan: false,
+    isEstimate: false,
+    hasFreshFruit: false,
+    hasFreshVegetables: false,
+  },
+  menuUid: "",
+  restaurantUid: "",
+  submittedByUserUid: "",
+  purchaseAt: [],
```

Comment:
Great work @johnmck95. I tested your work using the Xcode simulator on iOS devices (such as the iPhone 15), and I noticed a couple of differences that I believe are important to mention. Firstly, the navigation bar at the bottom of the page does not fit the screen size. Secondly, although it was displayed in your demo, the "Purchase At" subsection of the admin page was not visible.
<img width="721" alt="Screenshot 2024-02-13 at 10 41 14 AM" src="https://github.com/SENG480a-NutriDine/mobile/assets/92160564/1bad7de5-e6ea-44cd-9aac-f80d948542b1">

Reply:
Awesome, thanks for finding those! The `purchaseAt` is easy - forgot to push that lol (it's been pushed now)

The layout tabs getting cut off is unexpected. It is almost certainly coming from `app/app/(tabs)/DataSubmission.tsx`, where `style={{ paddingTop: 48, paddingBottom: 16, ...styles.container }}`, specifically the paddingBottom is likely insufficient. I'll try to get a fix up shortly, but i can't test on ios unfortunately.

Reply:
I pushed a change for the layout tabs, would you mind checking again, please?

Reply:
Sure, I'll let you know shortly

Reply:
the "Purchase At" subsection of the admin page is now included, but the issue with the navigation bar remains
![Uploading Screenshot 2024-02-13 at 12.16.03 PM.png…]()

Reply:
I tested your work using the Android Studio on an Android device and it works well.

Reply:
Hmmm that's no better at all, I would've expected at least an improvement. Would you mind playing with the styling in `app/app/(tabs)/DataSubmission.tsx`  `<SafeViewArea/>` component, please? I'm hoping it's some simple padding/margin/height that needs minor tweaking, and it might be quicker if you make that change and push a fix rather than our back and forth. 

I'll try again to find some way of setting up ios on my devices over reading break

Reply:
The DropDown placeholder text also looks unnaturally close to the left. I tried updating with the following, which from what i read should work, but no dice. I think that one is probably safe to leave until we get the big stuff done

```
       placeholder: {
            paddingLeft: 10,
            marginLeft: 10,
          },
```

Reply:
@johnmck95 setting the height to 80 in app/app/(tabs)/_layout.tsx will do the job for you.
<img width="1510" alt="Screenshot 2024-02-13 at 1 13 09 PM" src="https://github.com/SENG480a-NutriDine/mobile/assets/92160564/011e173b-6d8c-4c29-bc4a-4288e65eb735">
<img width="366" alt="Screenshot 2024-02-13 at 1 15 26 PM" src="https://github.com/SENG480a-NutriDine/mobile/assets/92160564/6b769b2b-b191-4398-a78d-6d276f8d9976">

Reply:
pushed! Thank you for looking into that